packages/fast-components-react-msft/src/button/button.props.ts

```diff
@@ -2,26 +2,18 @@ import * as React from "react";
 import { IButtonHandledProps as IBaseButtonHandledProps } from "@microsoft/fast-components-react-base";
 import { IManagedClasses, IMSFTButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
 
-export interface IButtonHandledProps extends IBaseButtonHandledProps {
-    /**
-     * The lightweight justified option
-     */
-    justified?: boolean;
-
-    /**
-     * The lightweight option
-     */
-    lightweight?: boolean;
-
-    /**
-     * The outline option
-     */
-    outline?: boolean;
+export enum Subtype {
```

Comment:
Subtype seems odd to me for button, because I expect that it would be a subset of the type attribute. This is really just specific to style. Can we revisit the naming on this?

Reply:
Yes we should @janechu and I noodled over it for awhile. We cant use style or type. I thought about ButtonStyle but Jane thought that was redundant... we could also do something like "appearance"

Reply:
+1 `appearance` could work

Reply:
I agree, I like appearance

Reply:
+ 1 - great suggestion.